_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -246,6 +246,16 @@ This `img` element is hidden with `visibility: hidden`.
 </div>
 ```
 
+#### Inapplicable Example 6
+
+This `img` element is [programmatically hidden][] with `content-visibility: hidden`.
+
+```html
+<div style="content-visibility: hidden">
+  <img src="/test-assets/shared/w3c-logo.png" />
+</div>
```

Comment:
Until this is better supported I don't think we should add this. This example fails in Firefox and Safari. I think its better to use hidden=until-found here. If you wanted to pass in Chromium you'd still need to support content-visibility, but the example wouldn't be a failure in browsers that don't support it yet. What's also neat specifically about using it in this rule is that there's nothing to be found here, so this couldn't become a visible image through some user action.

```suggestion
<div hidden="until-found">
  <img src="/test-assets/shared/w3c-logo.png" />
</div>
```

Reply:
`hidden="until-fonud"` also seems to have poor support: [MDN browser compatibility](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden#browser_compatibility).
I guess it's best to just park this until there is better support…